lib/datadog/ci/ext/environment.rb

```diff
@@ -257,7 +264,7 @@ def extract_jenkins(env)
           {
             Datadog::Ext::Git::TAG_BRANCH => branch,
             Datadog::Ext::Git::TAG_COMMIT_SHA => env['GIT_COMMIT'],
-            Datadog::Ext::Git::TAG_REPOSITORY_URL => env['GIT_URL'],
+            Datadog::Ext::Git::TAG_REPOSITORY_URL => env['GIT_URL'] || env['GIT_URL_1'],
```

Comment:
Minor: Should we also do `|| env['GIT_URL_2']`? Or is a "URL_1" always present if "URL_2" exists?

Reply:
that's correct, it's always present.